content/en/containers/cluster_agent/admission_controller.md

```diff
@@ -130,11 +130,29 @@ Finally, run the following commands:
 
 ### APM and DogStatsD
 
-To configure DogstatsD clients and APM tracers automatically, inject the environment variables `DD_AGENT_HOST` and `DD_ENTITY_ID` by using one of the following:
+To configure and inject APM tracers libraries and DogstatsD clients automatically:
 
 - Add the label `admission.datadoghq.com/enabled: "true"` to your pod.
 - Configure the Cluster Agent admission controller by setting `mutateUnlabelled` (or `DD_ADMISSION_CONTROLLER_MUTATE_UNLABELLED`, depending on your configuration method) to `true`.
 
+To opt-in your container for library injection, use Pod annotations inside your application's YAML file to specify which language tracers and should be injected, along with their versions.
+
+The annotations are a `key: value` pair in the following format:
+
+```yaml
+admission.datadoghq.com/<language>-tracer.version: <tracer-version>
+```
+
+Adding a this annotation means the tracer library for that language and version is injected into the containerized application. You can add several `<language>-tracer.version` annotations to inject multiple language tracers into one container.
+
+For example to inject the latest Java tracer and Ruby tracer version 1.4.0:
+
+```yaml
+annotations:
+    admission.datadoghq.com/java-tracer.version: "latest"
```

Comment:
We should warn on the risk of using latest (breaking change can happen on major versions)

Reply:
Ruby is not supported right now, let's put node instead.
Also, should we put these annotations in different sections? I don't think it's common that the same app uses java and node?
```suggestion
annotations:
    admission.datadoghq.com/java-lib.version: "latest"

annotations:
    admission.datadoghq.com/ruby-lib.version: "<version>"
```